src/containers/main-page/cv-form/components/fields/certifications/index.tsx

```diff
@@ -1,10 +1,25 @@
-import { memo } from 'react';
+import { memo, useEffect } from 'react';
+import { useQueryClient } from 'react-query';
+import CircularSpinner from '../../../../../../common/components/circular-spinner/circular-spinner';
+import AddProfiency from '../../../../../../common/components/add-pattern';
+import { useUserFromDb } from '../personal-information/lib/query-hooks';
+import AddedCertificatesList from './components/addedSertificates/AddedCertificatesList';
+import { QueryKey } from './lib/query-key';
+import { DbUser } from '../../../../../../common/models/User';
 
 const Certifications = function (): JSX.Element {
+  const { data, isLoading } = useUserFromDb();
+
+  if (isLoading) {
+    return <CircularSpinner size="large" color="primary" />;
+  }
   return (
-    <>
-      Certificates Form
-    </>
+    <AddProfiency
+      collection={data?.certificates}
+      title="Certificate"
+    >
+      <AddedCertificatesList certificates={data?.certificates || []} />
```

Comment:
It is better to repalce array length checking to the place where you iteating the array

Reply:
I am doing checking in the AddedCertificatesList component. (!!certificates.length && certificates.map......). I need to pass empty array if I get data.certificates === undefined. Else I can't use map in undefined.